config/IpCfg.hpp

```diff
@@ -0,0 +1,29 @@
+// ======================================================================
+// \title  IpCfg.hpp
+// \author mstarch
+// \brief  hpp file for SocketIpDriver component implementation class
+//
+// \copyright
+// Copyright 2009-2015, by the California Institute of Technology.
+// ALL RIGHTS RESERVED.  United States Government Sponsorship
+// acknowledged.
+//
+// ======================================================================
+
+#ifndef REF_IPCFG_HPP
+#define REF_IPCFG_HPP
+
+enum IpCfg {
+    SOCKET_SEND_TIMEOUT_SECONDS = 1,       // Seconds component of timeout
```

Comment:
Could we specify that this is the send only timeout?

Reply:
The name has "SEND" in it.  Is that what you are referring to?

Reply:
Ah, I missed that, I was mostly looking at the comment. Looks good!